src/pages/CropSelector/CropCalendarView/RenderCrops.jsx

```diff
@@ -21,13 +21,6 @@ import '../../../styles/cropCalendarViewComponent.scss';
 import { updateSelectedCropIds } from '../../../reduxStore/cropSlice';
 import { myCropListLocation, snackHandler } from '../../../reduxStore/sharedSlice';
 import { setSaveHistory } from '../../../reduxStore/userSlice';
-import PSATooltip from '../../../components/PSAComponents/PSATooltip';
-
-const tooltipContent = () => (
-  <AcUnit
-    sx={{ color: 'white', backgroundColor: '#80D0FF', borderRadius: '5px' }}
-  />
-);
 
 const tooltipRatingContent = (crop, goal, councilShorthandRedux) => (
```

Comment:
There are a few `tooltipContent` left (including a few renamed to other like this), can you put all of them into the tooltip? You can use search in vscode for `tooltipContent` for all of them.